content/html/concepts/tags/terms/applet/applet.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: '<applet>'
+Description: 'Used to embed Java applets in HTML documents but is no longer supported.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Tags'
+  - 'Obsolete'
+CatalogContent:
+  - 'learn-html'
+  - 'learn-java'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<applet>` tag was used to embed Java applets into an HTML page. It has since been deprecated and removed from the [HTML specification](https://html.spec.whatwg.org/). This tag should no longer be used as browsers no longer support it.
```

Comment:
```suggestion
The `<applet>` tag was used to embed Java applets into an HTML page. It has since been deprecated and removed from the [HTML specification](https://html.spec.whatwg.org/). This tag should not be used since browsers no longer support it.
```